### PR TITLE
Booking rules

### PR DESCRIPTION
**Pull request by HristoSLV**

### CODE REVIEW

**Review by ignatIgnatov — COMMENTED** [review]
Looks good, BUT big conflict with booking-crud PropertyTestHelper coming up...

File: src/test/java/ludogorie_soft/reservations_platform_api/controller/PropertyControllerIntegrationTest.java [hunk 1]
```diff
@@ -66,6 +64,7 @@ void setUp() {
         property.setPetAllowed(true);
         property.setPetRules("Cats only");
         property.setPrice(200);
+        property.setPropertyRules("No loud noises after 10 PM");
```
Comment (emiliyank) [inline src/test/java/ludogorie_soft/reservations_platform_api/controller/PropertyControllerIntegrationTest.java:67]: Can we use the `createDefaultProperty()` from the `PropertyTestHelper` here?

File: src/test/java/ludogorie_soft/reservations_platform_api/controller/PropertyControllerIntegrationTest.java [hunk 2]
```diff
@@ -78,16 +77,18 @@ void createProperty_ShouldReturnCreatedProperty() throws Exception {
         requestDto.setPetAllowed(false);
         requestDto.setPrice(150);
         requestDto.setPetRules("No pets allowed");
+        requestDto.setPropertyRules("Guests should maintain cleanliness");
```
Comment (emiliyank) [inline src/test/java/ludogorie_soft/reservations_platform_api/controller/PropertyControllerIntegrationTest.java:80]: Can we use the `createDefaultPropertyRequestDto()` here?

Comment (HristoSLV) [reply src/test/java/ludogorie_soft/reservations_platform_api/controller/PropertyControllerIntegrationTest.java]: PropertyControllerIntegrationTest has been updated to use the PropertyTestHelper class

**Pull request merged**